components/scream/src/physics/shoc/atmosphere_macrophysics.hpp

```diff
@@ -313,13 +355,44 @@ class SHOCMacrophysics : public scream::AtmosphereProcess
         const Real  phis_i(phis(i));
         T_mid(i,k) = PF::calculate_temperature_from_dse(dse_ik,z_mid_ik,phis_i);
       });
+
+      /*--------------------------------------------------------------------------------
+       *DRY-TO-WET MMRs:
+       *-----------------
+       *Since the host model (or AD) expects wet mixing ratios, we need to convert dry
+       *mixing ratios from SHOC to wet mixing ratios except for qv[which will be converted
+       *in the following "parallel for" after all the other tracers] and TKE [which is
+       already in wet mmr].
+       *---------------------------------------------------------------------------------
+       */
+
+      Kokkos::parallel_for(Kokkos::TeamThreadRange(team, num_qtracers-2), [&] (const Int& idx) {
```

Comment:
Same comment here as the loop in the preprocess.

Reply:
Fixed this the same way (added an additional parallel-for)